lib/Cake/Test/Case/Console/Command/TestShellTest.php

```diff
@@ -48,7 +48,12 @@ class TestShellTest extends CakeTestCase {
  *
  * @return void
  */
-	public function setUp() {
+	public function setUp() : void {
+		// This test can not be run with PHPUnit 9+.
```

Comment:
Maybe in that case we could just remove this test? Very likely we will never run it again, unless you have some idea how to decuple from `PHPUnit_Runner_StandardTestSuiteLoader`.

For now let's keep comment and skip test :+1:

Reply:
Yes, I agree with removing it.:+1:
I only skipped it because my focus was only on getting the test to pass.

Reply:
I will take care of it after merge. Resolving.